packages/visualizations/app/styles/navi-visualizations/visualizations/table.less

```diff
@@ -89,7 +89,6 @@
 
     &__number-format-dropdown {
```

Comment:
Why do we have three places where styles for `number-format-dropdown` can be defined?